lib/msf/core/auxiliary/scanner.rb

```diff
@@ -299,6 +299,36 @@ def scanner_show_progress
   end
 end
 
+def add_delay_jitter(_delay, _jitter)
+  # Introduce the delay
+  delay_value = _delay.to_i
```

Comment:
Here and elsewhere, there is no need to `to_i` these things because the `datastore` options from which they are derived will ensure that they are integers.